install/source/index.markdown

```diff
@@ -53,9 +61,48 @@ Pull down required repositories and build from within the root directory of your
     wstool update -t src
     rosdep install -y --from-paths src --ignore-src --rosdistro ${ROS_DISTRO}
     catkin config --extend /opt/ros/${ROS_DISTRO} --cmake-args -DCMAKE_BUILD_TYPE=Release
+
+## Optional: Disable Packages from Building
+
+MoveIt is a large project and the defailt compile time can take ~30 minutes.
+If you would like to reduce your compile time, you can disable certain unneeded packages from being built.
+Be careful not to disable packages that are needed dependency for others.
+There are several approaches to disabling packages:
```

Comment:
```suggestion
You can exclude packages from a build in several ways:
```

Reply:
You're splitting hairs here with this rewording... disabling vs excluding... they mean the same thing here

Reply:
Just ignore it if you don't agree.

It's a suggestion.